### DISCUSSION TITLE
How to set up opening message in Chatflow mode of the ChatBot?

### DISCUSSION THREAD

**Discussion by guzi5618** [langgenius/dify#5049] [Suggestion]
I can't find a widget to setup the opening message in chatflow mode. Please help me.

**Comment by patryk20120** [comment]
You can find it under `Featrures`
![image](https://github.com/langgenius/dify/assets/9155931/3f74002a-0a80-408c-877e-a6832623bc90)

and then
![image](https://github.com/langgenius/dify/assets/9155931/3325be46-bb16-42b8-9f3c-4e6a33e71459)
